modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -1517,6 +1518,119 @@ func (r Routing) RegisterV2(mux *mux.Router, oidcKubeConfEndpoint bool, oidcCfg
 		Handler(r.getSeedOverview())
 }
 
+// swagger:route GET /api/v2/projects/{project_id}/providers/gcp/disktypes project listProjectGCPDiskTypes
+//
+//	List disktypes for a given project
+//
+//	Consumes:
+//	- application/json
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: GCPDiskTypeList
+func (r Routing) listProjectGCPDiskTypes() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(provider.ListProjectGCPDiskTypes(r.presetProvider, r.userInfoGetter)),
+		provider.DecodeProjectGCPDisktypes,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+// swagger:route GET /api/v2/projects/{project_id}/providers/gcp/subnetworks gcp listProjectGCPSubnetworks
+//
+// Lists available GCP subnetworks
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: GCPSubnetworkList
+func (r Routing) listProjectGCPSubnetworks() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(provider.ListProjectGCPSubnetworks(r.presetProvider, r.userInfoGetter, r.seedsGetter)),
+		provider.DecodeProjectGCPSubnetworks,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+// swagger:route GET /api/v2/projects/{project_id}/providers/gcp/networks gcp listProjectGCPNetworks
+//
+// Lists available GCP networks
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: GCPNetworkList
+func (r Routing) listProjectGCPNetworks() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+		)(provider.ListProjectGCPNetworks(r.presetProvider, r.userInfoGetter)),
+		provider.DecodeGCPCommonReq,
```

Comment:
`GCPCommonReq` doesn't include project information, so this probably needs its own request type with project information (don't forget to annotate it with `// swagger:parameters listProjectGCPNetworks`).